### DISCUSSION TITLE
Thumbnails on viewers

### DISCUSSION THREAD

**Discussion by evripidis** [aaronwmorris/indi-allsky#745] [Ideas]
I think it would be a very handy addition to have the ability to view our Images and Timelapses at once on a single webpage with thumbnails and date-time title.

At this very moment I find myself on an endless mouse click, trying to review my images for lightnings (yes the weather is really bad….)

**Comment by aaronwmorris** [comment]
This is not very obvious, but you can use the left and right arrow keys to cycle through the images in the image viewer.

**Reply by aaronwmorris** [reply]
Merged #747 to adjust hours using the up/down arrow keys.  Also added instructions to the view.

**Comment by evripidis** [comment]
Oh I see! That was an Easter Egg :-) Yes it works, and certainly it helps a lot! Thank you!


Just for the record, my idea was a view like this, I do not know how easy or not is to add as an alternative way.
![Screenshot 2023-05-09 at 21 33 29](https://github.com/aaronwmorris/indi-allsky/assets/216827/b90a6022-f5f4-425e-8cf2-876363b7086a)

**Reply by aaronwmorris** [reply]
I understand what you are asking for.  I will experiment with this when I get some time.

I remember playing with some gallery options like what is in TJ's all sky.  I just remember it having considerable trouble considering how many images are generated with an all sky camera.  3000 images in one night is not unusual.

**Reply by evripidis** [reply]
Oh ok ok I get that. So maybe probably will be more useful and less resource consuming to populate a webpage with time-lapse or keograms etc. maybe even only on a monthly basis? 
If we do have day and night time-lapse enabled, I think that loading 60 thumbnails will be OK for most of our systems.
